### PR TITLE
Add READMEs for branch management tools

### PR DESCRIPTION
**Pull request by petr-muller**

### CODE REVIEW

File: cmd/config-brancher/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Once the tool selects the set of configurations to operate on, it does one of two actions: Mirror the configuration out,
+or bump the configuration files.
+
+### Mirroring configuration
+
+In the mirror configuration mode, it copies the development branch config to all branches for the provided
+`--future-release` values, not changing the configuration for the dev branch and making sure that the release branch for
+the version that matches that in the dev branch has a disabled promotion stanza to ensure only one branch feeds a
+release ImageStream.
```
Comment (bbguimaraes) [inline cmd/config-brancher/README.md:39]: This was a bit hard to follow, how about:

```suggestion
In the mirror configuration mode, it copies the development branch configuration to all branches for the provided
`--future-release` values. The development branch version is not changed. Since this results in both the development
branch and one of the release branches promoting to the same release ImageStream, promotion is disabled in the release
branch for the version which matches that in the promotion stanza of the development branch. This ensures only one
branch feeds that release ImageStream.
```

Comment (petr-muller) [reply cmd/config-brancher/README.md]: Like that!
